app/models/cim_base_storage_extent.rb

```diff
@@ -10,10 +10,7 @@ def self._virtual_reflections
   end
 
   def self.find(*args)
-    options = args.extract_options!
-    options[:conditions] = CimStorageExtent.merge_conditions(options[:conditions], :id => base_storage_extent_ids)
-    args << options
-    CimStorageExtent.find(*args)
+    CimStorageExtent.where(:id => base_storage_ext_ids).find(*args)
```

Comment:
ugh, this is ungood. I left the [calls to] `find` methods on AAArMs alone, on the basis they were obviously implementing their own thing... I didn't realise some are just passing them along to a different, actual-AR, model. :confused:

Anyway, this is at least an improvement. The fact this wasn't/isn't showing a deprecation in the tests, however, suggests it's not covered at all, FYI.